src/store/modules/keycodes/quantum.js

```diff
@@ -8,6 +8,46 @@ function makeLT(layer) {
     title: `kc on tap, switch to layer ${layer} while held`
   };
 }
+// make a One-Shot Mod Keycode Definition
+const osmLookup = {
+  MOD_LSFT: ['LSft', 'Left Shift'],
```

Comment:
does this need to be an array anymore? you're aren't using the 2nd value

Reply:
I'm not using the second value because I never figured out how to check that `mod` is valid. I still intend to use this data.